PxWeb/Code/PxDatabase/Builders/MenuBuilder.cs

```diff
@@ -125,13 +132,49 @@ public void EndBuild(string path)
 
         }
 
+        //Crude sorting, from ChatGPT.
+        static void SortElements(XElement element)
+        {
+            if (! element.HasElements)
+               { return; }
+            var dsfsd = element.Name;
+
+            var sortedChildren = element.Elements()
+                                        .OrderBy(el => GetOrderByValue(el))
```

Comment:
Can we not do a sort instead of order by that will sort the original list of elements? That way we would not have to rearrange the ourself